spec/requests/articles_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe "Articles API", type: :request do
```

Comment:
🎱 
```suggestion
RSpec.describe "Articles API" do
```
It looks like your tests are set up to [infer the test type](https://github.com/danfrenette/a-different-shade-of-blue/blob/develop/spec/rails_helper.rb#L71). If you're going to specify them explicitly everywhere I'd remove them from the `rails_helper`. If you're going to leave the config in, I wouldn't specify the test unless it's in an "unexpected" place.

Reply:
Made a ticket to remove these in a separate unit of work since they're in every spec as of right now.